samples/cellular/nrf_cloud_rest_fota/README.rst

```diff
@@ -101,30 +100,32 @@ Building and running
 
 .. include:: /includes/build_and_run_ns.txt
 
-The configuration file for this sample is located in :file:`samples/cellular/nrf_cloud_rest_fota`.
+The configuration files for this sample are located in :file:`samples/cellular/nrf_cloud_rest_fota`.
 See :ref:`configure_application` for information on how to configure the parameters.
 
-To create a FOTA test version of this sample, add the following parameter to your build command:
+To create a **FOTA** test version of this sample, add the following parameter to your build command:
```

Comment:
```suggestion
To create a FOTA test version of this sample, add the following parameter to your build command:
```
the text is made bold for UI, PCB elements and pin names. These terms could be kept as a plain text.